integration_tests/storage_interactions.spec.ts

```diff
@@ -208,4 +207,38 @@ describe('Users storing data', () => {
     await expect(unauthorizedStorage.openFileByUuid(file?.uuid || ''))
       .to.eventually.be.rejectedWith(FileNotFoundError);
   }).timeout(TestsDefaultTimeout);
+
+  it('should subscribe to textile events', async (done) => {
+    const { user } = await authenticateAnonymousUser();
+    const txtContent = 'Some manual text should be in the file';
+
+    const storage = new UserStorage(user);
+    await storage.initListener();
+
+    const ee = await storage.txlSubscribe();
+
+    const eventData = new Promise<TxlSubscribeEventData>((resolve) => {
+      ee.once('data', (d:TxlSubscribeEventData) => d);
+    });
+
+    const uploadResponse = await storage.addItems({
+      bucket: 'personal',
+      files: [
+        {
+          path: 'top.txt',
+          data: txtContent,
+          mimeType: 'plain/text',
+        },
+        {
+          path: 'subfolder/inner.txt',
+          data: 'some other stuffs',
+          mimeType: 'plain/text',
+        },
+      ],
+    });
+
+    const data = await eventData;
+    expect(data.bucketName).to.equal('personal');
+    done();
+  });
```

Comment:
Also, you should add `.timeout(TestsDefaultTimeout)` to the end of this like other tests

Reply:
Fixed